cmd/version/version.go

```diff
@@ -0,0 +1,58 @@
+package version
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	adminclient "github.com/flyteorg/flyteidl/clients/go/admin"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+	"github.com/flyteorg/flytestdlib/version"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// Long descriptions are whitespace sensitive when generating docs using sphinx.
+const (
+	versionCmdShort = `Used for fetching flyte version`
+	versionCmdLong  = `
+Example version.
+::
+
+ bin/flytectl version
+`
+)
+
+// VersionCommand will return version of flyte
+func GetVersionCommand() *cobra.Command {
+	versionCmd := &cobra.Command{
+		Use:     "version",
+		Short:   versionCmdShort,
+		Aliases: []string{"versions"},
+		Long:    versionCmdLong,
+		Run: func(cmd *cobra.Command, args []string) {
+
+			ctx := context.Background()
+			adminClient, err := adminclient.InitializeAdminClientFromConfig(ctx)
+			if err != nil {
+				fmt.Printf("err %v:", err)
+				os.Exit(1)
+			}
+			v, err := adminClient.GetVersion(ctx, &admin.GetVersionRequest{})
+			if err != nil {
+				fmt.Printf("err %v:", err)
+				os.Exit(1)
+			}
+			version.LogBuildInformation("flytectl")
+			PrintVersion("flyteadmin", v)
+		},
+	}
+
+	return versionCmd
+}
+
+func PrintVersion(appName string, res *admin.GetVersionResponse) {
+	logrus.Info("------------------------------------------------------------------------")
```

Comment:
plz use `logger` from stdlib...

Reply:
@EngHabu I can make these changes....but it's a command....we can't add its output in the log. I found that flytestdlib version package only prints version in log(We need to set the log level to print output).